### DISCUSSION TITLE
School enrollment issue

### DISCUSSION THREAD

**Discussion by CapitaineColette** [community/community#86108] [GitHub Education] [answered]
### Hi everyone,

Hello,
Our students cannot be affiliated with our school because we do not provide our students with email addresses using our domain name. Are there any other solutions?
Thank you in advance for your help :)

**Comment by DTS-11** [comment, marked as answer]
Hi @CapitaineColette 

I don't think that there is another way to get students dev pack without a valid school email. You can try requesting them, but no other way in my knowledge.

**Reply by CapitaineColette** [reply]
Thanks :(
